packages/utils/src/__tests__/getAccountListWithAuthToken.spec.ts

```diff
@@ -0,0 +1,150 @@
+import getAccountListWithAuthToken from '../getAccountListWithAuthToken';
+import { AuthorizeResponse } from '@deriv/api-types';
+
+describe('getAccountListWithAuthToken', () => {
+    let accounts: NonNullable<AuthorizeResponse['authorize']>['account_list'];
+
+    beforeEach(() => {
+        localStorage.clear();
+
+        localStorage.setItem(
```

Comment:
would be good to trigger localStorage.clear() in afterEach to avoid leaking state to next tests (we already had state leakeages, the more tests we will have the bigger the risk and the more problems it can cause).

ideally, we should completely mock localStorage and avoid using "real" local storage whatsoever,